src/App/rootReducer.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
```

Comment:
Create a file called `store`. 

This file should 

1. create the store and have it be a default export (this will be helpful for testing the store in tests since you can import it)
2. I see no need to export the root reducer, you never directly use it in your app code (after my suggestions)